widget/toolbar.go

```diff
@@ -37,7 +40,12 @@ func (t *ToolbarAction) SetIcon(icon fyne.Resource) {
 
 // NewToolbarAction returns a new push button style ToolbarItem
 func NewToolbarAction(icon fyne.Resource, onActivated func()) *ToolbarAction {
-	return &ToolbarAction{icon, onActivated}
+	t := &ToolbarAction{
+		icon,
+		onActivated,
+		false,
+	}
+	return t
```

Comment:
I'd rather keep it the way it was returning the struct directly. There is no need to store it in a variable first. It is also a good idea to switch to named values given that you don't have to specify all field (false is the default value for bool).

```suggestion
	return &ToolbarAction{
		Icon: icon,
		OnActivated: onActivated,
	}
```